See
 http://www.iso.org/iso/en/prods-services/iso3166ma/03updates-on-iso-3166/nlv12-div.html

Add SERBIA, it has not yet been committed upstream.
Keep SERBIA AND MONTENEGRO for a transition phase.

Index: glibc-2.3.6/locale/iso-3166.def
===================================================================
--- glibc-2.3.6.orig/locale/iso-3166.def
+++ glibc-2.3.6/locale/iso-3166.def
@@ -197,6 +197,7 @@
 DEFINE_COUNTRY_CODE ("SAO TOME AND PRINCIPE", ST, STP, 678)
 DEFINE_COUNTRY_CODE ("SAUDI ARABIA", SA, SAU, 682)
 DEFINE_COUNTRY_CODE ("SENEGAL", SN, SEN, 686)
+DEFINE_COUNTRY_CODE ("SERBIA", RS, SRB, 688)
 DEFINE_COUNTRY_CODE ("SERBIA AND MONTENEGRO", CS, SCG, 891)
 DEFINE_COUNTRY_CODE ("SEYCHELLES", SC, SYC, 690)
 DEFINE_COUNTRY_CODE ("SIERRA LEONE", SL, SLE, 694)
